### ISSUE TITLE
Create new live widget "Ecommerce Realtime Map"

### ISSUE THREAD

**Issue by sgiehl** [matomo-org/matomo#12686]
Similar to the Realtime Map showing all visitors it would be nice feature for ecommerce websites to have a similar widget that only shows visits that ordered something.

Would do almost the same as Realtime Map, but fetches the visits segmented by `visitEcommerceStatus==orderedThenAbandonedCart,visitEcommerceStatus==ordered`.
It should maybe show the ecommerce details in tooltip as well

**Comment by tsteur** [comment]
I quickly implemented it but not sure how useful it is? Considering most users wouldn't have too many orders and not too much across the world or so that it is too useful? The regular map be more useful to see that is archived by default but then you wonder if it may be better for ecommerce sites to create a segment `visitEcommerceStatus==ordered` by default?

Could be maybe also done in a plugin or in a later release if really needed? Eventually we will support this also through Custom Reports I suppose. 

It's nice but for most ecommerce stores it might not be too useful?

**Comment by tsteur** [comment]
Also some users would probably prefer to see only orders and not abandoned cards etc

**Comment by mattab** [comment]
Could we do the real-time map behave like the standard real-time map, showing the last X~100 orders? this way it would look good even on low orders ecommerce sites, where the orders might span 1 week or more.

> Also some users would probably prefer to see only orders and not abandoned cards etc

Ideally below the map would be the control to switch and choose whether to show orders OR abandoned carts:
![Screenshot from 2020-01-10 14-36-27](https://user-images.githubusercontent.com/466765/72118642-a136a980-33b6-11ea-9e53-0cd0bf62700d.png)

**Comment by tsteur** [comment]
It would be doable probably with a lot more effort...  And the switch is not really userfriendly btw and hard to discover (as so many things).
 
I'm still doubting though it's usefulness just for a "nice to have" it might not be worth the time? And it could always be added later anyway since there are no breaking changes.

@mattab

**Comment by mattab** [comment]
yes that switch is very hard to fine... would be better to have it like it was before where we showed always the 2 options and labels on screen (before redesign)

no strong preference, just a nice to have

**Comment by tsteur** [comment]
I'll move it for now into 4.2 since it doesn't have any breaking change and can always be added at any time to ensure Matomo 4.0 won't take too long. Can still put it into backlog later.
